pkg/recovery/recover.go

```diff
@@ -80,11 +80,6 @@ type controlPlane struct {
 	daemonSets  v1beta1.DaemonSetList
 	deployments v1beta1.DeploymentList
 	secrets     v1.SecretList
-
-	// for self hosted etcd recovery if not nil
-	bootEtcd        *asset.Asset
```

Comment:
can we separate this out to another commit? want to keep this commit cleaner.

Reply:
split into two commits... still working on addressing the comments.